packages/main/test/pages/Avatar.html

```diff
@@ -163,6 +163,50 @@ <h3>Avatar - interactive</h3>
 		<ui5-input id="click-event-2"></ui5-input>
 	</section>
 
+	<section>
```

Comment:
Could you also add an example with a custom badge to see how it works?

Reply:
Done